src/Firehose.Web/Authors/SebastianJensen.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+    public class SebastianJensen : IAmACommunityMember
+    {
+        public string FirstName => "Sebastian";
+        public string LastName => "Jensen";
+        public string StateOrRegion => "Pforzheim, Germany";
+        public string EmailAddress => "apps@tsjdev-apps.de";
+        public string ShortBioOrTagLine => "Sebastian Jensen is Mobile .NET Developer and creates apps for UWP, iOS and Android.";
+        public Uri WebSite => new Uri("https://www.tsjdev-apps.de");
+        public string TwitterHandle => "tsjdevapps";
+        public string GitHubHandle => "tsjdev-apps";
+        public string GravatarHash => "d990a05a189c263901ca94367d3a50be";
+        public GeoPosition Position => new GeoPosition(48.892186, 8.694629);
+        public bool Filter(SyndicationItem item) => item.Categories.Any(cat => cat.Name.ToLowerInvariant() == "xamarin");
```

Comment:
Remove the filter. It is not hit anyways since you don't implement `IFilterMyBlogPosts`. We filter for xamarin in the title and categories anyways.